gene2phenotype_project/gene2phenotype_app/models.py

```diff
@@ -160,7 +160,7 @@ class LGDVariantTypeDescription(models.Model):
     id = models.AutoField(primary_key=True)
     lgd = models.ForeignKey("LocusGenotypeDisease", on_delete=models.PROTECT)
     publication = models.ForeignKey("Publication", on_delete=models.PROTECT, null=True)
-    description = models.CharField(max_length=250, null=False)
+    description = models.CharField(max_length=250, null=False, default='')
```

Comment:
Usually we don't want empty strings. Is there a specific reason for setting `default=''`?